auslib/db.py

```diff
@@ -2504,7 +2503,7 @@ def send_email(relayhost, port, username, password, to_addr, from_addr, table, s
 
 
 def make_change_notifier(relayhost, port, username, password, to_addr, from_addr, use_tls):
-    def bleet(table, type_, changed_by, query, transaction):
+    def bleet(table, type_, changed_by, query, transaction, *table_args):
```

Comment:
Maybe you can use kwargs, will not necessary guessing  the parameters indexes, just get by key if is in dict.

Reply:
ok lemmie see

Reply:
this has been implemented.. the args are now kwargs